src/Proto.Cluster/DefaultClusterContext.cs

```diff
@@ -21,7 +21,7 @@ public class DefaultClusterContext : IClusterContext
 
     private readonly PidCache _pidCache;
     private readonly ShouldThrottle _requestLogThrottle;
-    private readonly TaskClock _clock;
+    private readonly TaskClock? _clock;
```

Comment:
This is not used for dotnet 6 and greater? Perhaps just add a conditional block instead of making it nullable?

Reply:
Done